### PR TITLE
Fix help text for generatekey's output parameter

### LINKED ISSUE #1119: Option description text wrong
In the help documentation for `generatekey` the `output` help string includes a wrong value.

```
generatekey:
  Generate api key and hash for authentication.

Usage:
  dotnet-monitor generatekey [options]

Options:
  -o, --output <Cmd|Json|PowerShell|Shell|Text>    The format to display the output in. Valid values are Json, Text,
                                                   and EnvVar. [default: Json]
  -?, -h, --help                                   Show help and usage information
```

### PR DESCRIPTION
**Pull request by kelltrick**
Corrects an error in the `generatekey` command's `output` parameter help text. 

Old output of `dotnet monitor generatekey --help`:
```
generatekey:
  Generate api key and hash for authentication.

Usage:
  dotnet-monitor generatekey [options]

Options:
  -o, --output <Cmd|Json|PowerShell|Shell|Text>    The format to display the output in. Valid values are Json, Text,
                                                   and EnvVar. [default: Json]
  -?, -h, --help                                   Show help and usage information
```

New output of `dotnet monitor generatekey --help`:
```
generatekey:
  Generate api key and hash for authentication.

Usage:
  dotnet-monitor generatekey [options]

Options:
  -o, --output <Cmd|Json|PowerShell|Shell|Text>    The format to display the output in. This option is useful for
                                                   printing the information in a format to be copy/pasted into the
                                                   system of your choice. [default: Json]
  -?, -h, --help                                   Show help and usage information
```

Closes #1119

### CODE REVIEW

File: src/Tools/dotnet-monitor/Strings.resx [hunk 1]
```diff
@@ -352,7 +352,7 @@
     <comment>Gets the string to display in help that explains what the '--urls' option does.</comment>
   </data>
   <data name="HelpDescription_OutputFormat" xml:space="preserve">
-    <value>The format to display the output in. Valid values are Json, Text, and EnvVar.</value>
+    <value>The format to display the output in. This option is useful for printing the information in a format to be copy/pasted into the system of your choice.</value>
```
Comment (jander-msft) [inline src/Tools/dotnet-monitor/Strings.resx:355]: Feels a bit over explaining. How about replacing the entire description with something like `The output format of the API Key configuration.`

Comment (kelltrick) [reply src/Tools/dotnet-monitor/Strings.resx:355]: This is too specific. The intention is that this parameter can be a common parameter attached to any command that can write out information in multiple formats. 
Not that I expect this to ever be useful, but I intended this to be re-usable if we made `dotnet monitor config show --output PowerShell` write out the configuration block as environment variables.

Comment (jander-msft) [reply src/Tools/dotnet-monitor/Strings.resx:355]: Keep the help text as specific as it is used right now and don't over generalize for something we may or may not do in the future.

Comment (shirhatti) [reply src/Tools/dotnet-monitor/Strings.resx:355]: I agree with Justin in that this seems over explained.
My 🌶️ take is that I prefer the original

Comment (jander-msft) [reply src/Tools/dotnet-monitor/Strings.resx:355]: We can always change it again later for the new usages of the text.

Comment (kelltrick) [reply src/Tools/dotnet-monitor/Strings.resx]: Updated to take @jander-msft 's suggestion.

**Review by jander-msft — APPROVED** [review]

**Pull request merged**